src/cookieSync.js

```diff
@@ -93,7 +119,51 @@ function ajax(url, callback, data, options = {}) {
   }
 }
 
-// Send empty data to receive cookie sync status for all prebid server adapters.
-// In next phase we will read placement id's from query param and will only get cookie sync status of bidders participating in auction
-var data = '{}';
-ajax(ENDPOINT, process, data);
\ No newline at end of file
+function getUrlParam(paramName) {
+  if (paramName && paramName.length > 0) {
+    var sPageURL = window.location.search.substring(1);
+    if (sPageURL) {
+      var sURLVariables = sPageURL.split('&');
+      for (var i = 0; i < sURLVariables.length; i++) {
+        var sParameterName = sURLVariables[i].split('=');
+        if (sParameterName && sParameterName.length == 2 && sParameterName[0] == paramName) {
+          return sParameterName[1];
+        }
+      }
+    }
+  }
+}
+
+function getBidders() {
+  var bidders = getUrlParam("bidders");
+  if (bidders && bidders.length > 0) {
```

Comment:
if bidders param is not passed then the code will not execute any pixel.
We should execute all pixels if bidders param is not passed.

Reply:
Yeah, there was typo in if condition brackets.
Done